screens/Payment.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import Toolbar from './../components/Toolbar';
+
+export default class App extends React.Component {
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.orangeBox}/>
+            <Toolbar/>
+            <Text style = {styles.header}>Thank U for Shopping!</Text>
```

Comment:
Input `Text` components where the actual name will show up. For the PR, I would recommend actually showing some dummy data (a fake username, venmo account, etc.)

Reply:
@miaseiner Thanks for putting in the dummy data. But they should be in their own Text component. Having a separate Text component makes it easier in the future inject data from the database